src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1330,29 +1607,35 @@ private static GencodeFuncotationBuilder createGencodeFuncotationBuilderWithTriv
                                                                                                         final GencodeGtfGeneFeature gtfFeature,
                                                                                                         final GencodeGtfTranscriptFeature transcript) {
 
+         //TODO: Add gtfFeature.getGeneType() as an annotation field in the GencodeFuncotation - Issue #4408
+
          final GencodeFuncotationBuilder gencodeFuncotationBuilder = new GencodeFuncotationBuilder();
          final Strand strand = Strand.decode(transcript.getGenomicStrand().toString());
 
-         gencodeFuncotationBuilder.setRefAlleleAndStrand(variant.getReference(), strand)
+         gencodeFuncotationBuilder.setRefAllele(variant.getReference())
+                 .setStrand(strand)
                  .setHugoSymbol(gtfFeature.getGeneName())
                  .setNcbiBuild(gtfFeature.getUcscGenomeVersion())
                  .setChromosome(gtfFeature.getChromosomeName())
-                 .setStart(variant.getStart());
+                 .setStart(variant.getStart())
+                 .setGeneTranscriptType(gtfFeature.getTranscriptType());
 
          // The end position is inclusive, so we need to make sure we don't double-count the start position (so we subtract 1):
          gencodeFuncotationBuilder.setEnd(variant.getStart() + altAllele.length() - 1)
                  .setVariantType(getVariantType(variant.getReference(), altAllele))
-                 .setTumorSeqAllele1(altAllele.getBaseString())
+                 .setTumorSeqAllele1(variant.getReference().getBaseString())
```

Comment:
There is already a `refAllele` field.

I think it might make sense just to remove the `tumorSeqAllele1` field and only output it on serialization, since it's just the ref allele.